sdk/storage/storage-blob/src/BlockBlobClient.ts

```diff
@@ -81,6 +87,14 @@ export interface BlockBlobUploadOptions {
    * @memberof BlockBlobUploadOptions
    */
   progress?: (progress: TransferProgressEvent) => void;
+  /**
+   * Customer Provided Key Info.
+   *
+   * @type {Models.CpkInfo}
+   * @memberof BlockBlobUploadOptions
+   */
+  customerProvidedKey?: Models.CpkInfo;
+  tier?: BlockBlobTier | string;
```

Comment:
Minor: add the @type and @memberof info, I find these are added for track 2, would better to make this consistent if this is convention for track 2.

Reply:
Added